Resources/doc/filter/pager.rst

```diff
@@ -89,3 +83,20 @@ View data returned by this filter to be used in template:
 +-------------------------+--------------------------------------------------+
 | getPagerService()       | Returns pager service to be used in template     |
 +-------------------------+--------------------------------------------------+
+| getTags()               | Lists all tags specified at filter configuration |
++-------------------------+--------------------------------------------------+
+| hasTag($tag)            | Checks if filter has the specific tag            |
++-------------------------+--------------------------------------------------+
+
+Template variables
+------------------
+
+To add pagination in twig template add this:
+
+.. code-block:: twig
+
+    {{ ongr_paginate(filter_manager.filters.list_pager.getPager(), 'ongr_search_page') }}
```

Comment:
`ongr_search_page ` is route name. Please name it something like `currect_route_name` or smth like that.